versions.yaml

```
@@ -232,8 +232,8 @@ externals:
 
   coco-guest-components:
     description: "Provides attested key unwrapping for image decryption"
-    url: "https://github.com/confidential-containers/guest-components/"
-    version: "0a06ef241190780840fbb0542e51b198f1f72b0b"
+    url: "https://github.com/Xynnn007/guest-components"
+    version: "4e800d72a23cbb9df38a6e5484733b2519630594"
```

```suggestion
    # Pending support in guest-components: https://github.com/confidential-containers/guest-components/pull/995
    url: "https://github.com/Xynnn007/guest-components"
    version: "4e800d72a23cbb9df38a6e5484733b2519630594"
```

... just to add context for reviewers.

---

Oh. Iet me add some more explanation